src/testing/jest/jest-runner.ts

```diff
@@ -13,6 +13,7 @@ export async function runJest(config: d.ValidatedConfig, env: d.E2EProcessEnv) {
     const emulateConfigs = getEmulateConfigs(config.testing, config.flags);
     env.__STENCIL_EMULATE_CONFIGS__ = JSON.stringify(emulateConfigs);
     env.__STENCIL_ENV__ = JSON.stringify(config.env);
+    env.__STENCIL_TRANSPILE_PATHS__ = !!config.transformAliasedImportPaths;
```

Comment:
this is admittedly funky but I didn't see another clean way to get a Stencil config value into the Jest preprocessor, and it seems like we do this already for some things.

Anywho - basically the intent is just to get the value of `transformAliasedImportPaths` into the Jest context so we can set the corresponding option correctly on the options object passed in to the transpilation functions.